### PR TITLE
Update CS-RIN-RU-ENHANCED-external.user.js

### PR DESCRIPTION
**Pull request by Altansar69**
Return default tag + change author, update url etc...

### CODE REVIEW

**Comment by DogancanYr** [comment]
> I don't know if we should have the [Cracked] tag, maybe something like [See on CS.RIN] because the [Cracked] is a bit misleading.

All the time, games that can't be cracked are marked as not cracked. Everything else is cracked. That seems fine to me.

**Comment by Altansar69** [comment]
I agree with DogancanYr. It's very rare for games not to be cracked and not to have an uncracked tag. It must be less than 1% of cs.rin games. That's why I think it's a shame not to write carcked when it's most likely the case.

**Comment by DogancanYr** [comment]
It will be difficult for those who do not understand coding and for new users to understand what is cracked and what is not, and the cracked article gives more happiness. Because it reminds you to use something paid for free.

**Comment by DogancanYr** [comment]
Userscrips are there to make things easier, not to make them harder.

File: CS-RIN-RU-ENHANCED-external.user.js [hunk 1]
```diff
@@ -3,18 +3,22 @@
...
 // @name:pt         CS.RIN.RU Melhorado (Externo)
 // @name:tr         Genişletilmiş CS.RIN.RU (Ek)
-// @namespace       Altansar
-// @version         1.1.5
+// @namespace       https://github.com/Altansar69/CS.RIN.RU-Enhanced-external
+// @version         1.1.6
 // @description     Everything that concerns CS.RIN.RU - Steam Underground Community but does not act on the site.
 // @description:fr  Tout ce qui concerne CS.RIN.RU - Steam Underground Community mais qui n'agit pas sur le site.
 // @description:pt  W.I.P.
 // @name:tr         CS.RIN.RU sitesini ilgilendiren her şey - Steam Underground Topluluğu ancak sitede faaliyet göstermemektedir.
```
[the remark this answers is not in the document]
Comment (Altansar69) [reply CS-RIN-RU-ENHANCED-external.user.js:11]: True

**Comment by Altansar69** [comment]
> Then why not leave it as blank like topics normally are? We already have the tag to show the user that the topic is _not_ on CS.RIN, so we don't necessarily need to add a default tag. We can also have some text like the "Copied!" text when we click the link button in CS.RIN Enhanced, but this one would show when the script has found the topic for a visual cue that the link has been loaded.

I think it's better to display something

**Comment by Altansar69** [comment]
after not necessarily the sentence I put if you have another idea, I agree

**Comment by DogancanYr** [comment]
> after not necessarily the sentence I put if you have another idea, I agree

Can't anything be done about changing text colors?
I made a suggestion earlier, but you rejected it because he forces the colors to change.

**Comment by DogancanYr** [comment]
Just cracked word golden, uncracked word bright red.

**Comment by SubZeroPL** [comment]
Just my 5 cents: I'd also remove the word [Cracked] and only leave [Not cracked] as that info is imho more important and also 100% correct (there can be some games that for example were not cracked in latest version, so they are not marked with Not cracked, but marking them as Cracked is also a bit misleading).
As for text colors

> Just cracked word golden, uncracked word bright red.

That is not a good idea, it works for you but can be problematic for others with other forum/system colors etc.
Current solution is optimal in that regard that it takes into consideration each user's color situation and sets color texts accordingly. The only other option would be to add some kind of configuration that can override this default behavior so we do not make script unusable for someone because we hardcode default colors.

**Comment by Altansar69** [comment]
On some games on cs.rin it is specified in the tag that which version is cracker. Unfortunately, this is not very common.

I agree with removing the cracked (by default) that you don't want. But I'm thinking of keeping the cracked tag. I'm in favor of just getting the tags from cs.rin and displaying them. It's ok for you ?

Exactly, the ideal solution would be to add options for manual configuration if we wish. But that would be a lot of work for so little.

**Comment by Altansar69** [comment]
I make a new commit adding the possibility of adding a default tag and modifying it. To do this, simply remove one of the first lines, which is a comment.

**Review by SubZeroPL — APPROVED** [review]

**Pull request merged**